Assets/Scripts/Player/PlayerController.cs

```diff
@@ -147,19 +153,32 @@ private void OnRightClick(InputAction.CallbackContext obj)
             if (Physics.Raycast(clickRay, out RaycastHit hit, 10000) && _attackLogic.Status == AttackLogic.AttackStatus.None)
             {
                 GameObject objectHit = hit.collider.gameObject;
+                Interactable interactable = hit.collider.GetComponent<Interactable>();
```

Comment:
objectHit.GetComp
It's just a minor thing, but it makes it a little bit faster I guess :]

Reply:
gotcha